### PR TITLE
Add release news for Shopware v6.5.7.0

### PR DESCRIPTION
**Pull request by kermie**
I made an example md for a possible structure in release note pages. Please ditch this PR if it is unsolicited or if the issue is already resolved.

### CODE REVIEW

File: src/6.5.6.x/6.5.6.0.md [hunk 1]
```diff
@@ -1 +1,64 @@
...
+
+## Abstract
+
+This minor version comes with improvements to HTML Sanitizer, logging, Admin Worker, updated UUID to version 7 and a lot of other features and closed bugs. See below for more details.
+
+## System requirements
+
+* tested on PHP v8.0, v8.1 and v8.2
+* tested on MySQL v8
+* from now on, Shopware needs the PHP library `bcmath` in order to function properly. You may install it with `$ apt install php-bcmath` on your linux machine. We already added it to our [system requirements](https://docs.shopware.com/en/shopware-6-en/first-steps/system-requirements) page.
```
Comment (shyim) [inline src/6.5.6.x/6.5.6.0.md:13]: why do we need bcmath?

File: src/6.5.7.x/6.5.7.0.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+### Deprecated LoggerFactory
+
+We deprecated the LoggerFactory for plugins for 6.6 as this does not consider project configuration of logging. Our suggested replacement is to use Monolog Channels, which works in any Shopware 6 version. Please also see the PayPal plugin as an example of our implementation:
+
+- https://github.com/shopware/SwagPayPal/blob/7.3.0/src/Resources/config/packages/monolog.yaml
+- https://github.com/shopware/SwagPayPal/blob/7.3.0/src/SwagPayPal.php#L200-L217
+
+### New low_priority queue
+
+We introduced a new queue `low_priority` additional to the already existing `async` queue. This queue can be used by plugins, to queue up messages which are not important in a short manner.
```
Comment (sushmangupta) [inline src/6.5.7.x/6.5.7.0.md:58]:
```suggestion
We introduced a new queue, `low_priority`, in addition to the already existing `async` queue. This queue can be used by plugins, to queue up messages which are not important in a short manner.
```

Comment (sushmangupta) [reply src/6.5.7.x/6.5.7.0.md:58]: What does in a short manner.

File: src/6.5.7.x/6.5.7.0.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+The existing `messenger:consume` or admin watcher will consume the new queue without configuration changes. With Shopware 6.6, it becomes required to setup an additional `messenger:consume` or just append the queue additionally to the existing one `messenger:consume async,low_priority`
+
+### Improved plugin:create
+
+The `plugin:create` command now can generate also:
```
Comment (sushmangupta) [inline src/6.5.7.x/6.5.7.0.md:64]:
```suggestion
The `plugin:create` command now can generate:
```

File: src/6.5.7.x/6.5.7.0.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+We are working right now also on a `dal:migration:create` to generate migration files automatically by comparing your entity definition with the current database schema. So stay tuned!
+
+### Soft delete of custom entities on App uninstall
+
+Custom Entities created by Apps will now be soft deleted, so that you can preserve the app data on uninstallation.
+
+### System update now uses the regular kernel
+
+The commands `system:update:prepare` and `system:update:finish` now use the regular kernel while updating Shopware. This fixes a lot of issues when plugins extended root components like Symfony Configuration or created themes.
```
Comment (sushmangupta) [inline src/6.5.7.x/6.5.7.0.md:80]:
```suggestion
The commands `system:update:prepare` and `system:update:finish` now use the regular kernel while updating Shopware. This fixes a lot of issues when plugins extend root components like Symfony Configuration or created themes.
```

File: src/6.5.7.x/6.5.7.0.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+* important and frequently requested bugfixes go here with a linked NEXT-number as well as a short description/headline
+* and another important one peeps longing for
+* and yet another one
+
+See all bugs fixed with this release: [link to a list of all bugs fixed with this release](link to a list of all bugs fixed with this release)
+
+## Credits
+
+Thanks to all thou diligent friends to help us making Shopware better and better with each pull request!
```
Comment (sushmangupta) [inline src/6.5.7.x/6.5.7.0.md:105]:
```suggestion
Thanks to all diligent friends for helping us make Shopware better and better with each pull request!
```

File: src/6.5.7.x/6.5.7.0.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+
+[TBD]
+
+* [maximilianruesch](https://github.com/maximilianruesch) for [NEXT-30599](https://github.com/shopware/shopware/blob/v6.5.6.0/changelog/release-6-5-6-0/2023-08-09-fix-overridden-super-calls-in-promise-chains-in-admin-components.md) and [NEXT-30427](https://github.com/shopware/shopware/blob/v6.5.6.0/changelog/release-6-5-6-0/2023-08-14-fix-sw-datepicker-to-load-refs-correctly-on-first-page-load.md)
+
+## More ressources:
```
Comment (sushmangupta) [inline src/6.5.7.x/6.5.7.0.md:111]:
```suggestion
## More resources:
```

File: src/6.5.7.x/6.5.7.0.md [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+
+* Study a detailled [diff on Github](https://github.com/shopware/shopware/compare/v6.5.7.0...v6.5.6.2) to the former version
```
Comment (sushmangupta) [inline src/6.5.7.x/6.5.7.0.md:113]:
```suggestion
* [Detailed diff on Github](https://github.com/shopware/shopware/compare/v6.5.7.0...v6.5.6.2) to the former version
```

File: src/6.5.7.x/6.5.7.0.md [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+* See the [changelog](https://github.com/shopware/shopware/blob/v6.5.7.0/CHANGELOG.md) for this version on GitHub
```
Comment (sushmangupta) [inline src/6.5.7.x/6.5.7.0.md:114]:
```suggestion
* [Changelog on GitHub](https://github.com/shopware/shopware/blob/v6.5.7.0/CHANGELOG.md) for this version.
```

File: src/6.5.7.x/6.5.7.0.md [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+* Read the marketing blog post at [https://www.shopware.com/en/news/shopware-6-release-news-november-2023/](https://www.shopware.com/en/news/shopware-6-release-news-november-2023/)
```
Comment (sushmangupta) [inline src/6.5.7.x/6.5.7.0.md:115]:
```suggestion
* [Marketing blog post](https://www.shopware.com/en/news/shopware-6-release-news-november-2023/)
```

File: src/6.5.7.x/6.5.7.0.md [hunk 10]
```diff
@@ -0,0 +1,121 @@
...
+* [Installation overview](https://developer.shopware.com/docs/guides/installation/)
+* [Update from a previous installation](https://developer.shopware.com/docs/guides/installation/template.html#update-shopware)
+
+## Get in touch
+
+Discuss about decisions, bugs you might stumble upon etc in our [community slack](https://slack.shopware.com). See you there ;)
```
Comment (sushmangupta) [inline src/6.5.7.x/6.5.7.0.md:121]:
```suggestion
Discuss about decisions, bugs you might stumble upon, etc in our [community slack](https://slack.shopware.com). See you there ;)
```

**Review by sushmangupta — CHANGES_REQUESTED** [review]

File: src/6.5.6.x/6.5.6.0.md (hunk 1 above)
Comment (kermie) [reply src/6.5.6.x/6.5.6.0.md]: that was an example only ;)

File: src/6.5.7.x/6.5.7.0.md [hunk 11]
```diff
@@ -0,0 +1,121 @@
+# Release notes Shopware 6.5.7.0
+
+## Abstract
```
[the remark this answers is not in the document]
Comment (kermie) [reply src/6.5.7.x/6.5.7.0.md:3]: It shall rather be an abstract than an introduction a la "this patch release contains two bug fixes, we didn't touch the storefront at all". We can also call it _synopsis_ if _abstract_ is misleading.

File: src/6.5.7.x/6.5.7.0.md [hunk 12]
```diff
@@ -0,0 +1,121 @@
+# Release notes Shopware 6.5.7.0
```
[the remark this answers is not in the document]
Comment (kermie) [reply src/6.5.7.x/6.5.7.0.md:1]: Right, they call it like that: https://www.shopware.com/en/news/shopware-6-release-news-october-2023/. But, to be very honest: as a developer, I am more interested in the version number, don't care the season when it was published.

[the remark this answers is not in the document]
Comment (bojanrajh) [reply src/6.5.7.x/6.5.7.0.md:1]: I'm with Marco on this one. On top of that, we have multiple releases every month - https://www.shopware.com/en/changelog/

**Pull request merged**